pro/src/screens/OfferType/__specs__/OfferTypeFetchEligibility.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, waitFor } from '@testing-library/react'
+import { createBrowserHistory } from 'history'
+import React from 'react'
+import { Router } from 'react-router'
+
+
+import { clickOnEducationalType, getEducationalOfferButton, getSubmitButton } from '../__tests-utils__/offerTypeUtils'
+import { setDefaultProps } from '../__tests-utils__/setDefaultProps'
+import OfferType, { IOfferTypeProps } from '../OfferType'
+
+const renderEACOfferCreation = (props: IOfferTypeProps) => {
+  const history = createBrowserHistory()
+  return render(
+    <Router history={history}>
+      <OfferType {...props} />
+    </Router>
+  )
+}
+
+describe('screens | OfferType', () => {
+  let props: IOfferTypeProps
+
+  beforeEach(() => {
+    props = setDefaultProps()
+  })
+  
+  it('should prevent submitting if offerer is not eligible to eac', async () => {
+    const testProps = { ...props, fetchIsOffererEligibleToEAC: jest.fn().mockRejectedValue({}) }
+    renderEACOfferCreation(testProps)
+
+    const educationalOfferButton = getEducationalOfferButton()
+    clickOnEducationalType(educationalOfferButton)
+
+    await waitFor(() => {
+      const submitButton = getSubmitButton()
+
+      expect(testProps.fetchIsOffererEligibleToEAC).toHaveBeenCalledWith()
+      expect(educationalOfferButton).toHaveProperty('disabled')
+      expect(submitButton).toHaveProperty('disabled')
+    })
```

Comment:
je crois que c'est pas autorisé d'avoir plusieurs expects dans un `waitWor` 

```suggestion
   await waitFor(() => expect(testProps.fetchIsOffererEligibleToEAC).toHaveBeenCalledWith())
   expect(educationalOfferButton).toHaveProperty('disabled')
   expect(submitButton).toHaveProperty('disabled')
```